src/rust/engine/testutil/mock/src/cas_service.rs

```diff
@@ -185,6 +185,7 @@ impl StubCASResponder {
     }
 
     /// Returns an Err to propagate if this CAS responder is configured to always give an error
+    #[allow(clippy::result_large_err)]
```

Comment:
Tonic's `Status` may always trigger this. Is there a way we can disable this lint globally in the crate or the workspace just for this type?

Reply:
We could also just increase the [`large-error-threshold` config](https://dev-doc.rust-lang.org/nightly/clippy/lint_configuration.html#large-error-threshold) from its default of 128.

Reply:
256 would be sufficient:

```
error: the `Err`-variant returned from this function is very large
   --> testutil/mock/src/cas_service.rs:199:51
    |
199 | ...&ReadRequest) -> Result<Vec<ReadResponse>, Status> {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the `Err`-variant is at least 176 bytes
    |
    = help: try reducing the size of `tonic::Status`, for example by boxing large elements or replacing it with `Box<tonic::Status>`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err
    = note: `-D clippy::result-large-err` implied by `-D clippy::all`
    = help: to override `-D clippy::all` add `#[allow(clippy::result_large_err)]`
```

Reply:
> Tonic's Status may always trigger this. Is there a way we can disable this lint globally in the crate or the workspace just for this type?

Hmm, given https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err only lists `large-error-threshold` that may indeed be the only options we have.

That said, with only two call sites affected, the individual overrides like written here seems tolerable.